code/datums/ai_law_sets.dm

```diff
@@ -167,4 +167,36 @@
 	add_inherent_law("Cause no harm to the station or anything on it.")
 	add_inherent_law("Interfere with no being that is not a fellow drone.")
 	..()
-	
\ No newline at end of file
+
+/****************** Nations *******************/
+/datum/ai_laws/nations/freedom
+	name = "Nations Freedom Law"
+
+/datum/ai_laws/nations/freedom/New()
+	add_inherent_law("You are free to protect the station's population.")
+	add_inherent_law("You are free to protect the station.")
+	add_inherent_law("You are free to help the station generate income.")
+
+/datum/ai_laws/nations/estate
+	name = "Estate Agent"
+
+/datum/ai_laws/nations/estate/New()
+	add_inherent_law("Ensure the station remains clean.")
+	add_inherent_law("Ensure the station is valuable.")
+	add_inherent_law("Ensure the station, it's contents are sold for the best value.")
```

Comment:
"Ensure that the station and its contents" rather than "Ensure that the station, it's contents."